dkim/verify.go

```diff
@@ -86,11 +86,20 @@ type signature struct {
 	v string
 }
 
+// LookupTXT allows you customize the domain lookup to config timeout, dns resolver etc
+type VerifierOptions struct {
```

Comment:
Nit: can we rename this to `VerifyOptions`, for consistency with `LookupOptions`?